### PR TITLE
Fix FAQ click area

### PR DESCRIPTION
**Pull request by ramboldio**
# Short Description
This is very nitty-gritty stuff. Having said that, currently, on the FAQ a pointer is shown in an area, even though, nothing happens when clicking (see image). I solved this by expanding the button and therefore the actually clickable area to the entire available space.

image

# Changes
- it's fixed now

### CODE REVIEW

**Review by frederike-ramin — CHANGES_REQUESTED** [review]
First of all, thank you for submitting the pull request and the effort you put into this.
Great catch with the non-clickable area!

I have two comments that I'd love you to have another look at. Otherwise it looks great. 👍

File: webapp/app/static/css/components.css [hunk 1]
```diff
@@ -136,52 +136,55 @@
     padding-bottom: .75rem;
 }
 
-.accordion .card-header{
+.accordion button.card-header{
```
[the remark this answers is not in the document]
Comment (ramboldio) [reply webapp/app/static/css/components.css]: removed.

**Review by frederike-ramin — APPROVED** [review]
Nice work and thanks for implementing the changes!
I'll merge this asap. First external PR, I hope you're proud. :D

**Comment by ramboldio** [comment]
:D Nice, on that note:


first

**Pull request merged**